packages/custom-field/theme/lumo/vaadin-custom-field-styles.js

```diff
@@ -83,6 +83,11 @@ const customField = css`
   :host([theme~='small'][has-label]) [part='error-message'] {
     font-size: var(--lumo-font-size-xxs);
   }
+
+  /* When custom-field is used with non-optimized components like rich-text-editor */
```

Comment:
Would suggest to not mention RTE here. We can make this more general by saying that this is for components without an outer margin.

Reply:
Ping @sissbruecker, done